bin/start-uc-cli-in-docker

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -ex;
+
+container_version="0.1.0"
+container_name="unitycatalog-cli"
+network_name="unitycatalog_network"
+network_details=$(docker network ls --format '{{json .}}' | jq -r --arg NETWORK_NAME "$network_name" 'select(.Name == $NETWORK_NAME) | .Name')
+container_details=$(docker container ls --format '{{json .}}' | jq -r --arg CONTAINER_NAME "$container_name" 'select(.Names == $CONTAINER_NAME) | .Names')
```

Comment:
This doesn't catch if the container is stopped. Minimally should change it to `ls -a` but you could simplify entirely by doing something like `docker container ls -a --filter "name=unitycatalog"`

Reply:
Fixed in [87c710b](https://github.com/unitycatalog/unitycatalog/pull/116/commits/87c710b0b124f2555d165593aa05446463c39da6)

Thanks for the suggestion.